### PR TITLE
Feature/version 2

### PR DESCRIPTION
**Pull request by breakdancingcat**
* New wayne.edu header 
* https://waynedotedu.tpondemand.com/entity/63905

### CODE REVIEW

**Comment by breakdancingcat** [comment]
@nickdenardis @robertvrabel @chrispelzer 
* I have updated this to the warrior strong school/college version
* It works on IE 11, Edge, FF, Safari, opera, etc
* Print out looks good
* Manually added this to alumni.wayne.dev to test (i left a branch for you to look at it if you want, 'header-test-do-not-merge' is the branch name)
* It went through the linter and is clean

File: src/header.html [hunk 1]
```diff
@@ -2,37 +2,42 @@
...
+                    <a href="https://login.wayne.edu">Login</a>
+                </div>
 
-            <form method="get" action="https://wayne.edu/search/">
-                <label for="q">Search:</label>
-                <input name="q" class="input-group-field" id="q" placeholder="Search..." type="text">
-                <button type="submit" class="button"><span class="search-icon"><span>Search</span></span></button>
-            </form>
+                <div class="wsusearchbar">
+                    <form method="get" action="/search/">
```
Comment (nickdenardis) [inline src/header.html]: For this version the action needs to be a fully qualified URL since this can be on any domain. `https://wayne.edu/search/`

**Comment by nickdenardis** [comment]
Three more deal breakers on this:

* Height is slightly different than wayne.edu (it should be consistent)
* The "Login" is bolder on this version than wayne.edu (it should not be bold)
* Kicking back the mobile view back to designers to allow Login and Search links to be in the header.

**Comment by nickdenardis** [comment]
@breakdancingcat This looks good to me! 👍

**Pull request merged**